src/components/PeopleTable.tsx

```diff
@@ -9,44 +177,88 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink
+                params={{ sort: sort('name'), order: order('name') }}
+                onClick={() => {
+                  searchParams.delete('order');
+                  handleSort(nameSort, setNameSort, 'name');
+                  setSexSort(Sort.notSorted);
+                  setBornSort(Sort.notSorted);
+                  setDiedSort(Sort.notSorted);
+                }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function). Fix all cases